### PR TITLE
#163081906 questions listing api route

### PR DESCRIPTION
**Pull request by jtad009**
####What does this PR do?
Meetup questions listing API endpoints

####Description of Task to be completed?

    create question model in nodejs
    persist meetup question  data to DB

####How should this be manually tested?

After cloning the repo, Cd in to the directory and run the npm install command and the npm start command after wards.
Using post man test all endpoint with the header

    key - Content-Type value: application/json

1. GET /api/v1/questions/

2.  PATCH /api/v1/questions/:id/up-vote

- using postman send the following data to upvote a meetup
replace :id with ID of question to upvotevote

3.  PATCH /api/v1/questions/:id/down-vote
- using postman send the following data to downvote a meetup
replace :id with ID of question to downvote
4. POST /api/v1/questions/
    using postman send the following data to create a new meetup question
    {
        "id": 4,
        "createdOn": "2018-09-12",
        "createdBy": 1,
        "meetup": 1,
        "title": "How to create data structures",
        "body": "How can i do this and that from time to time in the mix",
        "votes": 1
    }

    
####Any background context you want to provide?

A Json object was used to hold data in memory, this was used to add, edit, fillter questions and votes

####What are the relevant pivotal tracker stories?

#163081906
Screenshots (if appropriate)
Questions:

### CODE REVIEW

File: lib/routes/questions.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable radix */
+/* eslint-disable linebreak-style */
```
Comment (chukwuemekachm) [inline lib/routes/questions.js]: Please use es6 imports

File: test/app-spec.js [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+            return request(app).get('/api/v1/meetups/1').
+                set('Accept','application/json').
+                expect(200,{
+                    "status": 200,
+                    "data": [
+                        {
+                            "tags": [
+                                "Events",
+                                "Meetsups",
+                                "Andela",
```
Comment (chukwuemekachm) [inline test/app-spec.js]: Please use the specified 2 character spacing on this file.

File: lib/routes/questions.js (hunk 1 above)
Comment (jtad009) [reply lib/routes/questions.js]: es6 imports has been included. i just had to install babel

File: test/app-spec.js (hunk 2 above)
Comment (jtad009) [reply test/app-spec.js]: i have now used the 2 character spacing as advised i the code.

File: .idea/modules.xml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/questioner.iml" filepath="$PROJECT_DIR$/.idea/questioner.iml" />
+    </modules>
+  </component>
+</project>
```
Comment (chukwuemekachm) [inline .idea/modules.xml]: Please resolve all your EOL.

File: app-v1.js [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+});
+
+//create file server folder
+app.use(express.static('./public'));
+app.use(cors());
+/**
+ *add a meetup
+ */
+app.post('/api/v1/meetups/',function(request,response){
```
Comment (chukwuemekachm) [inline app-v1.js]: Please use 2 character spacing, on all files.

File: public/admin/index.html [hunk 5]
```diff
@@ -1,30 +1,46 @@
...
+  <title>Admin - Questioner</title>
+</head>
+<body>
+>>>>>>> ft-admin-UI-page-162799080
 <div class="sidebar">
     <img src="../images/bp-logo-home.png"    width="180px" class="logo" />
     <div> <img src="../images/avatar.jpg" class="user_passport"  /></div>
     <div class="navigation">
-       <a href="../profile/profile.html">View Profile</a>
+<<<<<<< HEAD
```
Comment (chukwuemekachm) [inline public/admin/index.html]: Please resolve this conflict properly.

File: public/css/login.css [hunk 6]
```diff
@@ -10,59 +10,68 @@ h1 {
...
-     justify-content: center;
-     height: 40vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 40vh;
 }
 .inner-div
 {
+<<<<<<< HEAD
```
Comment (chukwuemekachm) [inline public/css/login.css]: Kindly resolve all conflicts on your work.

**Pull request merged**